CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 * Removed deprecated `InstructionsBannerViewDelegate.didDragInstructionsBanner(_:)` method. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * Removed `NavigationViewController.origin` property, which was not used. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
 * Unified usage and configuration of `Navigator` instance under `NavigationProvider`. This also includes single entry point for configuring used tiles version and tile storage location. ([#2823](https://github.com/mapbox/mapbox-navigation-ios/pull/2823)) 
+* Added optional Predictive Caching mechanism which pre-fetches tiles which may become needed during navigation if app goes offline at some point. ([#2830](https://github.com/mapbox/mapbox-navigation-ios/pull/2830))
```

Comment:
This changelog entry should point out the specific changes in the API, such as renaming `NavigationOptions(styles:navigationService:voiceController:topBanner:bottomBanner:)` to `NavigationOptions(styles:navigationService:voiceController:topBanner:bottomBanner:predictiveCacheLocationOptions:)`.

Reply:
Updated